.github/workflows/ci.yml

```diff
@@ -193,6 +193,16 @@ jobs:
           vault_instance: ${{ env.VAULT_INSTANCE }}
           common_secrets: |
             SIGN_PLUGIN_ACCESS_POLICY_TOKEN=plugins/sign-plugin-access-policy-token:token
+            GITHUB_APP_ID=plugins-platform-bot-app:app-id
+            GITHUB_APP_PRIVATE_KEY=plugins-platform-bot-app:private-key
+
+      - name: Generate GitHub token
+        id: generate-github-token
+        uses: actions/create-github-app-token@v1.11.5
+        with:
+          app-id: ${{ env.GITHUB_APP_ID }}
+          private-key: ${{ env.GITHUB_APP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
```

Comment:
I suppose the provided `${{ secrets.GITHUB_TOKEN }}` (https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication)  isn't enough in this case?

Reply:
I also tried using the GitHub token `${{ secrets.GITHUB_TOKEN }}`, but unfortunately, the Go dependency download failed with an error message saying the repository was not found.

Reply:
Makes sense :+1: I'm marking this conversation as resolved then!